s2/point_vector_test.go

```diff
@@ -79,7 +74,7 @@ func TestPointVectorBasics(t *testing.T) {
 			t.Errorf("shape.Chain(%d).Length = %v, want %d", i, got, want)
 		}
 		edge := shape.Edge(i)
-		pt := randomPoint()
```

Comment:
We were testing a new random point and it still worked? Does this test need stronger conditions?